external/upstream/brianqc/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+find_package(BrianQC 1.1 CONFIG QUIET)
+
+if(${BrianQC_FOUND})
+    get_property(_loc TARGET BrianQC::static_wrapper PROPERTY LOCATION)
+    if(${BrianQC_DEV})
+        message(STATUS "${Cyan}Found BrianQC${ColourReset}: ${_loc} (found development version ${BrianQC_VERSION})")
+    else(${BrianQC_DEV})
+        message(STATUS "${Cyan}Found BrianQC${ColourReset}: ${_loc} (found installed version ${BrianQC_VERSION})")
+    endif(${BrianQC_DEV})
```

Comment:
Yes, the BrianQC_DEV is set in BrianQCConfig.cmake, and it is actually no longer really necessary, since it is only used to tweak the CMake message depending on the BrianQC version. I've also removed the superfluous conditions from the else() and endif().

Reply:
In fact, after reviewing the CMake, I realized that it is probably better to remove that variable altogether, since it served only a cosmetic purpose.